src/org/usfirst/frc4904/robot/commands/SetRampState.java

```diff
@@ -3,22 +3,18 @@
 
 import org.usfirst.frc4904.robot.RobotMap;
 import org.usfirst.frc4904.robot.subsystems.GearIO.RampState;
-import edu.wpi.first.wpilibj.command.Command;
 
-public class SetRampState extends Command {
+public class SetRampState extends OverridableCommand {
 	protected final RampState state;
+	public static boolean rampOverride;
 
 	public SetRampState(RampState state) {
 		this.state = state;
 	}
 
 	@Override
 	protected void initialize() {
+		override = SetRampState.rampOverride;
```

Comment:
we need the override to apply to all setRampStates but we cant make the override in overridable command static, or else it will apply to all overridable commands